pkgs/development/tools/ttfb/default.nix

```diff
@@ -1,40 +1,34 @@
 { darwin
 , fetchCrate
 , lib
-, openssl
-, pkg-config
 , rustPlatform
 , stdenv
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "ttfb";
-  version = "1.7.0";
+  version = "1.8.0";
 
   src = fetchCrate {
     inherit pname version;
-    hash = "sha256-GxjG8pyE2rY0h1dpAo+HRUbP31I5Pm4h1fAb6R7V+qU=";
+    hash = "sha256-UjZcdfOsSB2Pk3aRI1O4YUFk/nnO4dIw15XoKRzGkYk=";
   };
 
-  cargoHash = "sha256-YdbVtVKt0bKb1R5IQxf9J/0ZA3ZHH+oZ8ryX6f4cGsY=";
+  cargoHash = "sha256-NV+y9tE8l7lehGzaCiWFWTwAowjThGBEtzrCWB3TtaQ=";
 
   # The bin feature activates all dependencies of the binary. Otherwise,
   # only the library is build.
   buildFeatures = [ "bin" ];
 
-  nativeBuildInputs = [ pkg-config ];
-
-  buildInputs = [
-    openssl
-  ] ++ lib.optionals stdenv.isDarwin [
+  buildInputs = lib.optionals stdenv.isDarwin [
```

Comment:
@SuperSandro2000 I'm not sure if I need this now, as the binary also don't depend on openssl anymore (it uses rustls now). Advice?

Reply:
Remove it and see how that works. Ofborg should build it for darwin